sanic/app.py

```diff
@@ -27,7 +27,7 @@
 class Sanic:
 
     def __init__(self, name=None, router=None, error_handler=None,
-                 load_env=True, request_class=None,
+                 load_env=True, request_class=None, is_request_stream=False,
```

Comment:
I think it makes more sense to set `self.is_request_stream = True` in the `app.stream` method, if used. This would avoid having to initialize it here.